lua/swenv/api.lua

```diff
@@ -6,13 +6,34 @@ local ORIGINAL_PATH = vim.fn.getenv('PATH')
 
 local current_venv = nil
 
+local get_conda_base = function(conda_exe_path)
+  local parts = {}
+
+  for part in conda_exe_path:gmatch("[^/]+") do
+    table.insert(parts, part)
+  end
+  table.remove(parts) -- remove the last part
```

Comment:
couldn't `parent` from `plenary.path` be used here?

Reply:
yea, I wasn't aware of that. I updated it to use plenary, let me know if there's a better way to write it.